Azure/azure-cli
# [Security] Add adaptive_network_hardenings commands

**Description**  
[Security] Add Adaptive network hardeningss commands

Add Adaptive network hardenings commands, list of all possible traffic between resources for the subscription and location.

**Testing Guide**  
Users can get a list, single (show) of list of Adaptive Network Hardening resources:

az security adaptive_network_hardenings list --resource-group 'RG1' --resource-type 'virtualMachines' --resource-namespace 'Microsoft.Compute' --resource-name 'VM1'

az security adaptive_network_hardenings show --resource-group 'RG1' --resource-type 'virtualMachines' --resource-namespace 'Microsoft.Compute' --resource-name 'VM1' --adaptive-network-hardenings-resource-name 'default'

Tests are included, use:  azdev test security test_adaptive_network_hardenings --live
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [*] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [*] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).
